src/InteractsWithFaker.php

```diff
@@ -2,10 +2,13 @@
 
```

Comment:
Maybe we can now rename this file to just Faker?

Reply:
Done, I thought that would be better just wasn't sure. 👍